barretenberg/cpp/pil/vm2/context.pil

```diff
@@ -1,17 +1,79 @@
 // This is a virtual gadget, which is part of the execution trace.
 namespace execution;
 
```

Comment:
We need a skippable condition since this is a new relation even if it's virtual.

Reply:
ah good catch